### PR TITLE
get names consistent; associated doc updates based on test walkthrough

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees nodejs-ex changes for issue3358

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ One option is to use the Docker all-in-one launch as described in the [OpenShift
 
 ### The project ###
 
-If you don't have a project setup all ready, go ahead and take care of that
+After logging in (`oc login`, when any ID/Password combination should be fine), if you don't have a project setup all ready, go ahead and take care of that
```
Comment (bparees) [inline README.md]: s/when/with/ but more importantly any combination is only fine for the dev openshift environments we run, the one they're using will hopefully be more locked down.  so i'd just say to login with oc login.

File: README.md [hunk 2]
```diff
@@ -19,25 +19,27 @@ That's it, project has been created.  Though it would probably be good to set yo
...
 
-Now let's pull in the app source code from [GitHub repo](https://github.com/openshift/nodejs-ex) (fork if you like)
+Now let's pull in the app source code from [GitHub repo](https://github.com/openshift/nodejs-ex) (fork if you like).
 
 #### create ####
 
         $ oc new-app https://github.com/openshift/nodejs-ex -l name=myapp
         
-That should be it, `new-app` will take care of creating the right build configuration, deployment configuration and service definition.  Next you'll be able to kick off the build.  The -l flag will apply a label of "name=myapp" to all the resources created by new-app, for easy management later.
+That should be it, this form of `new-app` will take care of pulling down the necessary image streams, and then creating the right build configuration, deployment configuration and service definition.  Next you'll be able to kick off the build, though new-app will kick off a build once all required dependencies are confirmed.  The -l flag will apply a label of "name=myapp" to all the resources created by new-app, for easy management later.
```
Comment (bparees) [inline README.md]: it's not pulling down imagestreams, it's locating an appropriate builder image on dockerhub and creating an imagestream for that image.

File: README.md [hunk 3]
```diff
@@ -19,25 +19,27 @@ That's it, project has been created.  Though it would probably be good to set yo
...
 
 Note, you can follow along with the web console (located at https://ip-address:8443/console) to see what new resources have been created and watch the progress of the build and deployment.
 
 #### build ####
 
+If the build is not started, start one and follow it with:
```
Comment (bparees) [inline README.md]: If the build is not started (you can check by running "oc get builds"), start one and stream the logs with:

File: README.md [hunk 4]
```diff
@@ -19,25 +19,27 @@ That's it, project has been created.  Though it would probably be good to set yo
...
+
         $ oc start-build nodejs --follow
 
-You can alternatively leave off `--follow` and use `oc build-logs nodejs-n` where n is the number of the build (output of start-build).
+You can alternatively leave off `--follow` and use `oc build-logs nodejs-n` where n is the number of the build to track the output of the build.
 
 #### deploy ####
 
-happens automatically, to monitor its status either watch the web console or `oc get pods` to see when the pod is up.  Another helpful command is
+Happens automatically.  To monitor its status either watch the web console or execute `oc get pods` to see when the pod is up.  Another helpful command is
```
Comment (bparees) [inline README.md]: "Deployment happens automatically once the new application image is available."

File: README.md [hunk 5]
```diff
@@ -19,25 +19,27 @@ That's it, project has been created.  Though it would probably be good to set yo
...
-That should be it, `new-app` will take care of creating the right build configuration, deployment configuration and service definition.  Next you'll be able to kick off the build.  The -l flag will apply a label of "name=myapp" to all the resources created by new-app, for easy management later.
+That should be it, this form of `new-app` will take care of pulling down the necessary image streams, and then creating the right build configuration, deployment configuration and service definition.  Next you'll be able to kick off the build, though new-app will kick off a build once all required dependencies are confirmed.  The -l flag will apply a label of "name=myapp" to all the resources created by new-app, for easy management later.
 
 Note, you can follow along with the web console (located at https://ip-address:8443/console) to see what new resources have been created and watch the progress of the build and deployment.
 
 #### build ####
 
+If the build is not started, start one and follow it with:
+
         $ oc start-build nodejs --follow
```
Comment (bparees) [inline README.md]: this buildconfig name is wrong now (maybe it was before too, but it definitely is now)

File: README.md [hunk 6]
```diff
@@ -19,25 +19,27 @@ That's it, project has been created.  Though it would probably be good to set yo
...
 
-You can alternatively leave off `--follow` and use `oc build-logs nodejs-n` where n is the number of the build (output of start-build).
+You can alternatively leave off `--follow` and use `oc build-logs nodejs-n` where n is the number of the build to track the output of the build.
```
Comment (bparees) [inline README.md]: again wrong buildconfig/bulid name.

File: README.md (hunk 1 above)
Comment (gabemontero) [inline README.md]: change done ... will merge/commit/squash in the AM

File: README.md (hunk 2 above)
Comment (gabemontero) [inline README.md]: Changed that clause to:  "will locate an appropriate image on DockerHub, create an imagestream for that image, and then create the right build configuration, deployment configuration and service definition"

File: README.md (hunk 3 above)
Comment (gabemontero) [inline README.md]: Replaced .... merge/commit/squash in the AM

File: README.md (hunk 5 above)
Comment (gabemontero) [inline README.md]: Yep, it was wrong before (as the diff showed, I just added a blank line).  My observation was that with locating on DockerHub, the prefix was "nodejs-ex".  Updated.

File: README.md (hunk 6 above)
Comment (gabemontero) [inline README.md]: yep - updated

File: README.md (hunk 4 above)
Comment (gabemontero) [inline README.md]: Updated.

**Comment by gabemontero** [comment]
@bparees comments etc. incorporated for nodejs-ex

File: README.md [hunk 7]
```diff
@@ -19,25 +19,27 @@ That's it, project has been created.  Though it would probably be good to set yo
...
         
-That should be it, `new-app` will take care of creating the right build configuration, deployment configuration and service definition.  Next you'll be able to kick off the build.  The -l flag will apply a label of "name=myapp" to all the resources created by new-app, for easy management later.
+That should be it, this form of `new-app` will locate an appropriate image on DockerHub, create an ImageStream for that image, and then create the right build configuration, deployment configuration and service definition.  Next you'll be able to kick off the build, though new-app will kick off a build once all required dependencies are confirmed.  The -l flag will apply a label of "name=myapp" to all the resources created by new-app, for easy management later.
 
 Note, you can follow along with the web console (located at https://ip-address:8443/console) to see what new resources have been created and watch the progress of the build and deployment.
 
 #### build ####
 
-        $ oc start-build nodejs --follow
+If the build is not started (you can check by running "oc get builds"), start one and stream the logs with::
```
Comment (bparees) [inline README.md]: double colon here.

**Comment by bparees** [comment]
one more nit and then it looks good.

**Comment by gabemontero** [comment]
nit fixed - thx

**Comment by bparees** [comment]
lgtm.

**Pull request merged**